dockertest/output.py

```diff
@@ -737,11 +737,32 @@ def mustfail(cmdresult, failmsg=None):
         details = "%s\n%s" % (failmsg, cmdresult)
     if cmdresult is not None:
         OutputNotBad(cmdresult)
-    if cmdresult.exit_status == 0:
-        raise xceptions.DockerExecError("Unexpected zero exit code,"
-                                        " details: %s" % details)
+    # FIXME: temporary; remove once we no longer run on pre-1.10 docker
+    if not has_distinct_exit_codes():
+        expected_status = 1
+    if cmdresult.exit_status != expected_status:
+        raise xceptions.DockerExecError("Unexpected exit code %d; expected %d."
+                                        " Details: %s" % (cmdresult.exit_status,
+                                                          expected_status,
+                                                          details))
     return cmdresult
 
+HAS_DISTINCT_EXIT_CODES = None
+def has_distinct_exit_codes():
+    """
+    2016-03-23 **TEMPORARY** - for transition from docker-1.9 to 1.10
```

Comment:
FYI- These types of notes normally stick around /way/ longer than anyone ever intended, just look around :grinning:   These doc-strings will wind up in docker-autotest.readthedocs.org so short-and-sweet and tastefully-terse is usually the best way to go.